pkgs/ffi/lib/src/arena.dart

```diff
@@ -119,12 +119,12 @@ class Arena implements Allocator {
 R using<R>(R Function(Arena) computation,
     [Allocator wrappedAllocator = calloc]) {
   final arena = Arena(wrappedAllocator);
-  bool isAsync = false;
```

Comment:
I'm not sure if this is an improvement.

Maybe we should disable that lint until we've got alignment on the doc that we are not allowed to diverge from the lint set.

Reply:
> I'm not sure if this is an improvement.

Why not? It's obvious that `isAsync` is a `bool` both from the name and the initial value. So we don't gain any extra information from specifying the type explicitly. 

If the type helps with the clarity of the code, we can always ignore the lint for that line.

Reply:
Lets keep it as the default defined in the lint set for now.